microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -609,6 +610,36 @@ To enhance the Kids Mode experience, Microsoft Edge adds a safe keyword to the h
 Kids Mode does not collect data for personalization of the news feed or other Microsoft services.  You may not change the privacy settings for Kids Mode.  Other settings like Windows Defender SmartScreen and diagnostic data are configured according to the profile in which Kids Mode was launched.  For more information about diagnostic data about how you use the browser and Windows Defender SmartScreen, see the [Diagnostic Data](#diagnostic-data) and [SmartScreen](#smartscreen) sections.
 
 
+<!-- ====================================================================== -->
+## Magnify Image
+
+To use the Magnify Image feature, right-click an image and then select **Magnify image**, or press the **Ctrl** key twice while hovering over an image.  The Magnify Image feature retrieves the largest available version of the image from the website and displays it in a popup window.
+
+
+<!-- ------------------------------ -->
+#### AI-based image enhancement
+
+If your browser has the Web Machine Learning Neural Network (WebNN) API enabled, there's also an **AI enhance** button, that enables you to use AI to enhance the image.  To check whether WebNN is enabled, go to the `edge://gpu` page, and then in the **Graphics Feature Status** section, see whether **WebNN** says **Enabled**:
+
+![The status of the WebNN setting, shown in the edge://gpu page](./index-images/webnn-disabled-gpu.png)
+
+To change the WebNN setting, go to `edge://flags`, search for **Enables WebNN API**, and then select **Default**, **Enabled**, or **Disabled**.
+
+When you click the **AI enhance** button, AI-based image enhancement is done on your device by using a model that's downloaded from Microsoft servers.  Neither the image nor any personally identifiable information is sent to Microsoft.
+
+
+<!-- ------------------------------ -->
+#### Turning off the feature
+
+To turn off the Magnify Image feature, use the command-line flag `msEdgeImageMagnifyUI`.  For example, run Microsoft Edge from the command line using the `disable-features` command-line option, as follows:
+
+```
+msedge.exe --disable-features=msEdgeImageMagnifyUI
+```
```

Comment:
This should be removed. We don't usually tell people how to disable flags. They should be turning the feature on and off via settings. Also, this should be irrelevant once the feature is enabled by default.

```suggestion

```

Reply:
removed lines. @sjdallst2 - if i should restore anything, let me know